### DISCUSSION TITLE
[Feature] Supporting remote write protocols for Prometheus or VictoriaMetrics

### DISCUSSION THREAD

**Discussion by kalosmark** [apache/skywalking#11666] [Ideas]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Consider supporting remote write protocols for Prometheus or VictoriaMetrics, This feature can be implemented in the exporter module

### Are you willing to submit a pull request to implement this on your own?

- [X] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
Hi, if you plan to add new features, please prepare a complete proposal about how you implement.

**Reply by kalosmark** [reply]
Yes, currently I have implemented this feature in my current company and applied it in production environments, running well. Therefore, I would like to give this feature back to the community
   Because this is my first time submitting a feature, I don't know how to submit a proposal

**Comment by wu-sheng** [comment]
Please at least list following key parts in a general doc

- Feature Description
- Architecture Graph Describing the Relationship between your new proposal part and existing components
- Imported Dependencies
- Whether breaking configuration, storage structure, protocols
- General docs about when and how to use this

**Comment by wu-sheng** [comment]
The proposal process was just provided, you can read the doc now, https://skywalking.apache.org/docs/main/next/en/swip/readme/

And I have finished my first SWIP-1, https://github.com/apache/skywalking/discussions/11669
